wdl/transforms/draft3/src/test/scala/wdl/draft3/transforms/ast2wdlom/Ast2WdlomSpec.scala

```diff
@@ -0,0 +1,47 @@
+package wdl.draft3.transforms.ast2wdlom
+
+import java.util
+
+import cats.instances.either._
+import common.Checked
+import common.assertion.ErrorOrAssertions._
+import common.transforms.CheckedAtoB
+import org.scalatest.{FlatSpec, Matchers}
+import wdl.draft3.parser.WdlParser
+import wdl.draft3.parser.WdlParser.{ParseTree, SyntaxErrorFormatter}
+import wdl.draft3.transforms.parsing.WdlDraft3SyntaxErrorFormatter
+import wdl.model.draft3.elements._
+import wdl.transforms.base.ast2wdlom.GenericAstNode
+import scala.collection.JavaConverters._
+
+class Ast2WdlomSpec extends FlatSpec with Matchers {
+
+  val parser = new WdlParser()
+
+  def fromString[A](expression: String,
+                    parseFunction: (util.List[WdlParser.Terminal], SyntaxErrorFormatter) => ParseTree)
+                   (implicit converter: CheckedAtoB[GenericAstNode, A]): Checked[A] = {
+    val tokens = parser.lex(expression, "string")
+    val terminalMap = (tokens.asScala.toVector map {(_, expression)}).toMap
+    val parseTree = parseFunction(tokens, WdlDraft3SyntaxErrorFormatter(terminalMap))
+    (wrapAstNode andThen converter).run(parseTree.toAst)
+  }
+
+  it should "not parse the new as_map function" in {
```

Comment:
It *is* bad, only because you'd be able to end up with WDLs floating around with `version 1.0` at the top and `as_map`s in the middle - which would be non-portable to other engines.

Reply:
It's essentially the same reason we cant just add "just-for-us" syntax into our WDL parsers that openWDL doesn't necessarily want (or maybe actively un-wants)

Reply:
Excellent point, did not think about the portability meta-game